### ISSUE TITLE
Interhost connectivity broken between 2 hosts in a 3 host setup.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#9715]
Rancher versions: v1.6.8-rc4 with rancher/net:v0.11.8

Cattle environment with 3 hosts in the setup.
I had only couple of services running in this setup.
After 3 hours of installing rancher-server , I noticed that inter host connectivity is broken between 2 hosts in a 3 host setup.

Say the set up had hosts h1,h2 and h3.

Connectivity is broken between h1 and h2.
```swanctl --list-sas``` command from ipsec router form h1 and h2 has entries for only host h3.

But host h3 has entries for both h1 and h2.

**Comment by leodotcloud** [comment]
Summary of what happened:
Between two host H1, H2, H2 initiated the connection and by the time H1 got this packet, the ipsec config was not yet loaded on H1. Due to this `NO_PROPOSAL_CHOSEN` was sent by H1 to H2. And H2 didn't retry anymore.

```
00[CFG]   reading directory failed
00[CFG] loading attribute certificates from '/usr/local/etc/ipsec.d/acerts'
00[LIB] opening directory '/usr/local/etc/ipsec.d/acerts' failed: No such file or directory
00[CFG]   reading directory failed
00[CFG] loading crls from '/usr/local/etc/ipsec.d/crls'
00[LIB] opening directory '/usr/local/etc/ipsec.d/crls' failed: No such file or directory
00[CFG]   reading directory failed
00[CFG] loading secrets from '/usr/local/etc/ipsec.secrets'
00[LIB] loaded plugins: charon aesni aes des rc2 sha2 sha1 md5 random nonce x509 revocation constraints pubkey pkcs1 pkcs7 pkcs8 pkcs12 pgp dnskey sshkey pem fips-prf gmp xcbc cmac hmac gcm attr kernel-netlink resolve socket-default stroke vici updown xauth-generic
00[JOB] spawning 16 worker threads
08[NET] received packet: from 35.192.61.42[1024] to 10.42.197.73[500] (500 bytes)
08[ENC] parsed IKE_SA_INIT request 0 [ SA KE No N(NATD_S_IP) N(NATD_D_IP) N(FRAG_SUP) N(HASH_ALG) N(REDIR_SUP) ]
08[IKE] no IKE config found for 10.42.197.73...35.192.61.42, sending NO_PROPOSAL_CHOSEN
08[ENC] generating IKE_SA_INIT response 0 [ N(NO_PROP) ]
08[NET] sending packet: from 10.42.197.73[500] to 35.192.61.42[1024] (36 bytes)
09[CFG] loaded IKE shared key for: '%any'
06[CFG] loaded IKE shared key for: '130.211.131.174'
14[CFG] added vici connection: conn-130.211.131.174
16[CFG] loaded IKE shared key for: '10.42.197.73'
11[CFG] loaded IKE shared key for: '35.192.61.42'
09[CFG] added vici connection: conn-35.192.61.42
10[CFG] loaded IKE shared key for: '10.42.199.0'
11[CFG] loaded IKE shared key for: '10.42.162.10'
12[KNL] creating acquire job for policy 10.42.228.92/32[6/38830] === 10.42.148.9/32[6/42] with reqid {1234}
12[CFG] trap not found, unable to acquire reqid 1234
06[KNL] creating acquire job for policy 10.42.228.92/32[6/54332] === 10.42.92.142/32[6/42] with reqid {1234}
06[CFG] trap not found, unable to acquire reqid 1234
15[KNL] creating delete job for CHILD_SA ESP/0x00000000/130.211.131.174
15[JOB] CHILD_SA ESP/0x00000000/130.211.131.174 not found for delete
````

**Comment by leodotcloud** [comment]
Original issue: https://github.com/rancher/rancher/issues/8204

**Comment by leodotcloud** [comment]
Fixed in ipsec: 0.1.4 with image: rancher/net:v0.11.9

**Comment by sangeethah** [comment]
Have not seen this issue when testing with Rancher versions: v1.6.8-rc5 with rancher/net:v0.11.9
